README.md

```diff
@@ -343,7 +343,7 @@ Supported ethernet controllers:
 
 * __WizNet W5100__
 * __WizNet W5200__
-* __WizNet W5200__
+* __WizNet W5500__
```

Comment:
I think this change should be squashed into https://github.com/codebendercc/Ariadne-Bootloader/pull/27/commits/f07bc14783323283061df9b32103aa46b2de3c3b.